cpp/test/linalg/axpy.cu

```diff
@@ -100,18 +88,22 @@ class AxpyTest : public ::testing::TestWithParam<AxpyInputs<T>> {
     if ((params.incx > 1) && (params.incy > 1)) {
       axpy(handle,
            make_host_scalar_view<const T>(&params.alpha),
-           make_strided_device_vector_view<const T>(x.data(), params.len, params.incx),
-           make_strided_device_vector_view<T>(y.data(), params.len, params.incy));
+           make_device_vector_view<const T, IndexType, layout_stride>(
```

Comment:
Are we testing this with a device scalar view input as well?

Reply:
we weren't - but I've added tests for both the device/host scalar inputs in the last commit